sys/log/full/src/log_nmgr.c

```diff
@@ -164,6 +164,10 @@ log_nmgr_encode_entry(struct log *log, struct log_offset *log_offset,
      */
     g_err |= cbor_encoder_create_indef_byte_string(&rsp, &str_encoder);
     for (off = 0; off < len && !g_err; ) {
+        /* 
+         * Continue subsequent reads after inserting image hash after the 
+         * first read. 
+         */
```

Comment:
Is this comment accurate?  Maybe it is just out of date.  The image hash doesn't get inserted until after the body.